src/services/ExperianService.ts

```diff
@@ -101,40 +133,51 @@ export class ExperianService {
     	const storedToken = await this.getExperianToken();
     	const isValidToken = this.checkExperianToken(storedToken);
 
-    	if (isValidToken) {
+    	if (storedToken && isValidToken) {
+    		this.logger.info("Valid token found");
     		return storedToken;
-    	} else {		
-    		this.logger.info("requestExperianToken - no valid token - trying to generate new Experian token");
+
+    	} else {	
```

Comment:
```suggestion
    	} else {
```
Just a bit of dangling whitespace.
If we're returning ealy in the if block above, do we need this else?

Reply:
Sorry not sure I get you - we return early if we have a valid token, otherwise we perform the other steps, is that what you're asking about?